releasenotes/notes/security-compliance-agent-flare-command-87887fcc617424e1.yaml

```diff
@@ -0,0 +1,5 @@
+---
+enhancements:
```

Comment:
I think that's more a feature that an enhancements, no ?

Reply:
So we haven't fully shipped or announced these features, I think it's a rather small addition so "enhancement" might fit better here.

Reply:
If it's in the CHANGELOG it's announced publicly. `enhancement` is reserved to improvement on already existing feature (like adding an extra file to the flare). I would say this is either a `feature` or something we don't want to mention in the CHANGELOG (in case the security agent is not out yet).